packages/cli/src/environments/variables/variables.service.ee.ts

```diff
@@ -26,23 +23,25 @@ export class EEVariablesService extends VariablesService {
 		}
 	}
 
-	static async create(variable: Omit<Variables, 'id'>): Promise<Variables> {
+	async create(variable: Omit<Variables, 'id'>): Promise<Variables> {
 		if (!canCreateNewVariable(await this.getCount())) {
 			throw new VariablesLicenseError('Variables limit reached');
 		}
 		this.validateVariable(variable);
 
 		void Container.get(InternalHooks).onVariableCreated({ variable_type: variable.type });
-		return collections.Variables.save({
+		const saveResult = await collections.Variables.save({
 			...variable,
 			id: generateNanoId(),
 		});
+		await this.updateCache();
+		return saveResult;
 	}
 
-	static async update(id: string, variable: Omit<Variables, 'id'>): Promise<Variables> {
+	async update(id: string, variable: Omit<Variables, 'id'>): Promise<Variables> {
 		this.validateVariable(variable);
 		await collections.Variables.update(id, variable);
-
+		await this.updateCache();
```

Comment:
```suggestion
		void this.updateCache();
```

Reply:
not awaiting the cache update might save a few nanoseconds, but _potentially_ could turn into a race condition where the following change happens without the cache having been updated